website/components/comments.js

```diff
@@ -0,0 +1,15 @@
+import styles from "./comments.module.css";
+
+export default function MatthewComment({ children }) {
```

Comment:
Maybe change this to EditorComment?

Reply:
Yeah that's fair. Maybe Josh won't view this and then you will be the only one. Maybe you could make it Editor and have a parameter for name, but definitely not a blocking issue.